### PR TITLE
Issue 40 stream read test

### PR DESCRIPTION
**Pull request by dadlex**
Test PathRead with 3 events.

### CODE REVIEW

File: src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java [hunk 1]
```diff
@@ -126,7 +125,7 @@ public CommonTest ()
 
 		private CommonTest (final Config config)
 			throws OmgShootMyFootException {
-			super(
+			pravegaStorageDriver = new PravegaStorageDriver(
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java:128]: looks to be not needed

File: src/test/java/com/emc/mongoose/storage/driver/pravega/integration/DataOperationsTest.java [hunk 2]
```diff
@@ -43,9 +43,8 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-public class DataOperationsTest
-		extends PravegaStorageDriver<DataItem, DataOperation<DataItem>> {
-
+public class DataOperationsTest {
+	private final PravegaStorageDriver pravegaStorageDriver;
```
Comment (akurilov) [inline src/test/java/com/emc/mongoose/storage/driver/pravega/integration/DataOperationsTest.java:47]: looks to be not needed also

**Review by akurilov — CHANGES_REQUESTED** [review]

File: src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java (hunk 1 above)
Comment (dadlex) [reply src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java:128]: It's a stub that should test our driver in the future, if I understand correctly. So we should have an instance of driver. So we call its constructor. Can you please expand on what's wrong here?

Comment (akurilov) [reply src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java:128]: yep. then it's fine

**Review by akurilov — APPROVED** [review]

**Pull request merged**